### PR TITLE
EZP-29875: Facebook tag set focus issue

### PR DESCRIPTION
**Pull request by dew326**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29875
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js [hunk 1]
```diff
@@ -182,7 +184,7 @@ export default class EzBtnCustomTagUpdate extends EzWidgetButton {
             this.execCommand();
         }
 
-        const widget = this.getWidget();
+        const widget = this.getWidget() || this.widget;
```
Comment (sunpietro) [inline src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js]: Can you tell me more about it? You're setting `this.widget` value in the constructor. When the output of `this.getWidget()` will change its value?

Comment (dew326) [reply src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js]: The output of `this.getWidget()` will be null when we will lose focus in edit custom tag. This happens when we are editing custom tag and use cmd+a to select text in the input.

**Comment by lserwatka** [comment]
@micszo I merged it by mistake. Please report followups.

**Comment by micszo** [comment]
It works fine. 🎉

**Pull request merged**